Header header

uint64    down_speed            #in bits per second
uint64    up_speed              #in bits per second
float32   wifi_signal_strength  #in dB